### PR TITLE
Users models

### PR DESCRIPTION
**Pull request by KristianVega**

### CODE REVIEW

File: backend/src/apps/users/models/abstract_profile.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from src.caregivers.models import BaseModel
+from django.db import models
+
+
+class AbstractUser(BaseModel):
+    profile_image = models.TextField()
+    gender = models.CharField()  # pogledaj za enum
+    description = models.TextField()
+    type_user = models.TextField()  # caregivers && beneficial profile
```
Comment (KristianVega) [inline backend/src/apps/users/models/abstract_profile.py:9]: Ne treba nam u bazi

File: backend/src/apps/users/models/abstract_profile.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from src.caregivers.models import BaseModel
+from django.db import models
+
+
+class AbstractUser(BaseModel):
+    profile_image = models.TextField()
```
Comment (KristianVega) [inline backend/src/apps/users/models/abstract_profile.py:6]: Image file

File: backend/src/apps/users/models/abstract_profile.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    gender = models.CharField()  # pogledaj za enum
```
Comment (KristianVega) [inline backend/src/apps/users/models/abstract_profile.py:7]: Add verbose everywhere

File: backend/src/apps/users/models/beneficiary_profile.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    beneficiary_person = models.CharField(
+        verbose_name=_('beneficiary person'),
+        max_length=250
+    )
+    helping_period = models.CharField(
+        verbose_name=_('helping period'),
+        max_length=250
+    )
+    weekly_days = models.IntegerField()
+    daily_hours = models.IntegerField()
```
Comment (milos1302) [inline backend/src/apps/users/models/beneficiary_profile.py:20]: verbose_name missing in weekly_days & daily_hours

File: backend/src/apps/users/models/__init__.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+from apps.users.models.abstract_profile import AbstractProfile
+from apps.users.models.user import User
+from apps.users.models.caregiver_profile import CaregiverProfile
+from apps.users.models.beneficiary_profile import BeneficiaryProfile
```
Comment (milos1302) [inline backend/src/apps/users/models/__init__.py:4]: migrations missing for all of the above models

**Review by milos1302 — CHANGES_REQUESTED** [review]

**Review by milos1302 — APPROVED** [review]

**Pull request merged**